microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -45,136 +46,140 @@ This article uses five demo webpages, all sourced at https://github.com/Microsof
+The number below the new sidebar item shows the total size of the reachable JavaScript objects. To learn more about object sizes in the heap snapshot, see [Object sizes and distances](./memory-101.md#object-sizes-and-distances) in _Memory terminology_.
 
+The snapshot only displays the objects from the memory graph that are reachable from the global object. Taking a snapshot always starts with a garbage collection.
 
-<!-- ====================================================================== -->
-## Clear snapshots
 
-Click the **Clear all profiles** icon to remove snapshots (both from DevTools and any memory associated with the renderer process).
+<!-- ------------------------------ -->
+#### Take another snapshot
 
-![Remove snapshots](./heap-snapshots-images/heap-snapshots-all-hover-clear-all-profiles.png)
+To take another snapshot when one is already displayed in the **Memory** tool, click **Profiles** above the existing snapshot in the sidebar:
 
-Closing the DevTools window doesn't delete profiles from the memory associated with the renderer process.  When reopening DevTools, all previously taken snapshots reappear in the list of snapshots.
+![The Profiles button to take another snapshot](./heap-snapshots-images/take-another-snapshot.png)
 
 
-## Demo webpage: Example 3: Scattered objects
+<!-- ====================================================================== -->
+## Clear snapshots
 
-*  Open this demo example webpage: [Example 3: Scattered objects](https://microsoftedge.github.io/Demos/devtools-memory-heap-snapshot/example-03.html) in a new window or tab.  Profile it using the Heap Profiler.  A number of (object) item allocations are displayed.
+To clear all snapshots from the **Memory** tool, click the **Clear all profiles** (![The clear icon](./heap-snapshots-images/clear-icon.png)) icon:
 
-<!-- You can view the source files for the Heap Snapshots demo pages at the [MicrosoftEdge/Demos > devtools-memory-heap-snapshot](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-memory-heap-snapshot) repo folder. -->
+![Remove snapshots](./heap-snapshots-images/heap-snapshots-all-hover-clear-all-profiles.png)
 
 
 <!-- ====================================================================== -->
 ## View snapshots
 
-View snapshots from different perspectives for different tasks.
+Heap snapshots can be viewed in multiple different ways in the **Memory** tool. Each way corresponds to a different task:
 
-**Summary view** shows objects grouped by the constructor name.  Use it to hunt down objects (and the memory use) based on type grouped by constructor name.  **Summary view** is particularly helpful for **tracking down DOM leaks**.
+* The **Summary** view shows objects grouped by their constructor name.  Use it to find objects, and the memory they use, based on types grouped by constructor name. The **Summary** view is particularly helpful for **tracking down DOM leaks**.
```

Comment:
```suggestion
* The **Summary** view shows objects grouped by their constructor name.  Use the *Summary* view to find objects, and the memory they use, based on types grouped by constructor name. The **Summary** view is particularly helpful for tracking down DOM leaks.
```
try to make each sentence self-sufficient & meaningful without relying on "it" pointer indirection that requires resolving "it" by reading a different sentence.

Markdown POV: why bold tracking down DOM leaks?
Rendered POV: bolding might help, across multiple entries, like a table.  Finding: that bolding pattern is NOT used across items in list.

Rendered article section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots?branch=pr-en-us-3033#view-snapshots